modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageLocalServiceImpl.java

```diff
@@ -3334,6 +3334,10 @@ private WikiPage _updatePage(
 		page.setVersion(newVersion);
 		page.setMinorEdit(minorEdit);
 		page.setContent(content);
+		page.setSummary(summary);
+		page.setFormat(format);
+
+		page.setStatusByUserId(user.getUserId());
```

Comment:
Doesn't this line need to be sorted?  Not sure if the service.xml imposes a weird ordering. If this is fine here, you need to drop the blank line before the call.

Reply:
You are right

The correct order is

```
<column name="status" type="int" />
<column name="statusByUserId" type="long" />
<column name="statusByUserName" type="String" uad-anonymize-field-name="fullName" />
<column name="statusDate" type="Date" />
```

Thanks